pkg/controller/direct/bigtable/appprofile_mappings.go

```diff
@@ -25,7 +25,10 @@ func BigtableAppProfileSpec_FromProto(mapCtx *direct.MapContext, in *pb.AppProfi
 		return nil
 	}
 	out := &krmv1beta1.BigtableAppProfileSpec{}
+
 	// MISSING: Name
+	out.ResourceID = direct.LazyPtr(in.GetName())
```

Comment:
I don't think `resourceID` is the fully qualified name of the resource, it's just the last part `bigtableappprofile-${uniqueId}`. Can we handle this field in the controller? Currently it does not appear to be working, since `resourceID` is still missing from the generated golden object file.